app/controllers/mentor/base_controller.rb

```diff
@@ -0,0 +1,9 @@
+class Mentor::BaseController < ApplicationController
+  before_action :authenticate_user!
+  before_action -> { require_role 'mentor' }
+  # before_action :set_breadcrumbs
+
+  # def set_breadcrumbs
```

Comment:
It's copied over from somewhere else and I was not sure yet if I could use it for the not yet fully implemented show view - but will probably delete it. Thanx for pointing out!